src/common/password.cpp

```diff
@@ -56,8 +56,6 @@ namespace
 
   bool read_from_tty(epee::wipeable_string& pass, bool hide_input)
   {
-    static constexpr const char BACKSPACE = 8;
```

Comment:
I have no idea why someone chose to do this instead of using `\b`

Reply:
Most likely for symmetry with the Unix routine (which used BACKSPACE = 127, ASCII DEL). I'd suggest preserving this line. Although I would have named them both DELETE, not BACKSPACE.